mkdocs_monorepo_plugin/parser.py

```diff
@@ -180,6 +180,9 @@ def getNav(self):
         return self._prependAliasToNavLinks(self.getAlias(), self.navYaml["nav"])
 
     def _prependAliasToNavLinks(self, alias, nav):
+        # Creating a set of protocols monorepo should not touch
+        protocolSet = set(["http://", "https://"])
```

Comment:
Shouldn't this technically support other protocols such as `mailto:` or custom URL identifiers? Maybe a better approach is to identify if it is a relative link / markdown file, otherwise assume it is an absolute link.

Reply:
I think starting identifying relative urls is more complex/ error prone. I currently have no use-case for mailto: of other protocols I think if there is a need is is easy enough to extend the set to 
protocolSet = set(["http://", "https://"], "mailto://")